Description: Use a local copy of object.inv from doc.python.org, instead of downloading it each time from the internet

Index: python-numpy-1.4.1/doc/source/conf.py
===================================================================
--- python-numpy-1.4.1.orig/doc/source/conf.py	2010-07-13 00:25:45.059911225 +0200
+++ python-numpy-1.4.1/doc/source/conf.py	2010-07-13 00:26:47.816913315 +0200
@@ -205,7 +205,7 @@
 # -----------------------------------------------------------------------------
 # Intersphinx configuration
 # -----------------------------------------------------------------------------
-intersphinx_mapping = {'http://docs.python.org/dev': None}
+intersphinx_mapping = {'http://docs.python.org/dev': '../../debian/python.org_objects.inv'}
 
 
 # -----------------------------------------------------------------------------
